src/mono/wasm/Wasm.Build.Tests/Common/ProjectInfo.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+namespace Wasm.Build.Tests;
+
+public record ProjectInfo(
+    string Configuration,
```

Comment:
I think `Configuration` and `AOT` should not be here. I believe those should be just arguments for build (and possibly run). But for asset copying they are not relevant (and might be changed during test)